srcpkgs/godot/template

```diff
@@ -42,15 +40,38 @@ pre_build() {
 	export CXXFLAGS=" $CXXFLAGS "
 }
 
+do_build() {
+	for tools in yes no; do
+		for target in release release_debug; do
+			for platform in x11 server; do
+				# tools provides editor functionality and we don't need it without debug capabilities
+				if [[ $tools != yes || $target != release ]]; then
+					scons ${makejobs} ${make_build_args} CCFLAGS="$CFLAGS" LINKFLAGS="$LDFLAGS" tools=${tools} target=${target} platform=${platform}
+				fi
+			done
+		done
+	done
```

Comment:
This triple product with some combinations excluded is too clever by half. Unroll this in a sensible way so it's obvious what combinations are being built. Define a helper function for reuse if appropriate.

Also, the scons line is so long it's illegible, so add some judicious wraps.